<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChemicalStatus</class>
 <widget class="QDialog" name="ChemicalStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Оцінка хімічного становища</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>528</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Зона можливого хімічного забруднення (ЗМХЗ), км*км</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zone_line_edit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Прогнозована зона хімічного забруднення, км*км</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="area_line_edit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Глибина прогнозованої зони хімічного забруднення (Гпзхз), км</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="depth_line_edit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Ширина прогнозованої зони хімічного забруднення (Шпзхз), км</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="width_line_edit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>249</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Вертикальна стійкість повітря:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dvsa_line_edit">
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Кутова зона (Ф):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="angle_line_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
